src/main/java/seedu/duke/Storage.java

```diff
@@ -0,0 +1,58 @@
+package seedu.duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Represents a storage, stores and read data from given file location
+ */
+public class Storage {
+    protected String dataFileName;
+
+    /**
+     * Instantiates a new Storage object
+     */
+    public Storage(String filePath) {
+        this.dataFileName = filePath;
+    }
+
+    /**
+     * Writes the contents of a given TaskList to a specified file
+     *
+     * @param tasks the TaskList containing Task to be written to file
+     */
+    public void writeToFile(TaskList tasks) throws IOException {
+        //structure: command|1 (1 for mark, 0 for unmark)
```

Comment:
I like how the comment is added for clarity and the SLAP principles followed in the following lines. Good job!